examples/polars/notebook.ipynb

```diff
@@ -0,0 +1,435 @@
+       "<ellipse fill=\"none\" stroke=\"black\" cx=\"179.97\" cy=\"-306\" rx=\"39.58\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"179.97\" y=\"-300.95\" font-family=\"Times,serif\" font-size=\"14.00\">base_df</text>\n",
+       "</g>\n",
+       "<!-- base_df&#45;&gt;signups -->\n",
+       "<g id=\"edge1\" class=\"edge\">\n",
+       "<title>base_df&#45;&gt;signups</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M169.54,-288.41C164.33,-280.13 157.92,-269.92 152.1,-260.66\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"154.6,-259.07 146.32,-252.47 148.68,-262.8 154.6,-259.07\"/>\n",
+       "</g>\n",
+       "<!-- base_df&#45;&gt;spend -->\n",
+       "<g id=\"edge2\" class=\"edge\">\n",
+       "<title>base_df&#45;&gt;spend</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M192.26,-288.76C198.75,-280.19 206.85,-269.49 214.11,-259.9\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"217.49,-262.23 220.74,-252.15 211.91,-258.01 217.49,-262.23\"/>\n",
+       "</g>\n",
+       "<!-- base_df_location -->\n",
+       "<g id=\"node7\" class=\"node\">\n",
+       "<title>base_df_location</title>\n",
+       "<ellipse fill=\"none\" stroke=\"black\" stroke-dasharray=\"5,2\" cx=\"179.97\" cy=\"-378\" rx=\"98.44\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"179.97\" y=\"-372.95\" font-family=\"Times,serif\" font-size=\"14.00\">Input: base_df_location</text>\n",
+       "</g>\n",
+       "<!-- base_df_location&#45;&gt;base_df -->\n",
+       "<g id=\"edge7\" class=\"edge\">\n",
+       "<title>base_df_location&#45;&gt;base_df</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M179.97,-359.7C179.97,-352.24 179.97,-343.32 179.97,-334.97\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"183.47,-335.1 179.97,-325.1 176.47,-335.1 183.47,-335.1\"/>\n",
+       "</g>\n",
+       "<!-- spend_zero_mean&#45;&gt;spend_zero_mean_unit_variance -->\n",
+       "<g id=\"edge4\" class=\"edge\">\n",
+       "<title>spend_zero_mean&#45;&gt;spend_zero_mean_unit_variance</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M358.5,-72.41C345.75,-63.39 329.76,-52.09 315.79,-42.21\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"318.13,-38.87 307.95,-35.96 314.09,-44.59 318.13,-38.87\"/>\n",
+       "</g>\n",
+       "</g>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<graphviz.graphs.Digraph at 0x107555590>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Execute the driver.\n",
+    "\n",
+    "df = dr.execute(output_columns)\n",
+    "print(df)\n",
+    "\n",
+    "# To visualize do `pip install \"sf-hamilton[visualization]\"` if you want these to work\n",
+    "dr.visualize_execution(output_columns, './polars', {\"format\": \"png\"})\n",
+    "dr.display_all_functions('./my_full_dag.dot')"
```

Comment:
This can be simplified to this:
```suggestion
    "dr.visualize_execution(output_columns)\n",
```
Also only need one. Jupyter only displays the last one.